core/src/test/resources/api/coproduct/one_of/int_discriminator_with_mapping.yaml

```diff
@@ -21,6 +21,9 @@ components:
         - $ref: "#/components/schemas/Organization"
       discriminator:
         propertyName: name
+        mapping:
+          1: "#/components/schemas/Person"
+          2: "#/components/schemas/Organization"
```

Comment:
Int discriminator property without explicit mapping is not supported. So we changed the test to have an explicit mapping.